integrations/marketo-v2/lib/index.js

```diff
@@ -141,6 +177,15 @@ Marketo.prototype.page = function(page) {
   });
 };
 
+Marketo.prototype.setupAndSubmitForm = function(traits, form) {
+  form.addHiddenFields(traits, form);
+  // Do not remove this callback. This ensures there are no page refreshes after the form is submitted.
+  form.onSuccess(function() {
```

Comment:
Add an onFailure callback to not refresh in that scenario either

Reply:
No callback available https://developers.marketo.com/javascript-api/forms/api-reference/ :|